tools/mapping-tester/preparemeshes.py

```diff
@@ -111,6 +111,14 @@ def main(argv):
         print('Warning: outdir "{}" already exisits.'.format(outdir))
     meshdir = os.path.join(outdir, "meshes")
     function = setup["general"]["function"]
+    algorithm = setup["general"].get("partitioning", "meshfree")
+
+    if "partitioning" in setup["general"]:
+        print(f"Using partitioning algorithm {algorithm}")
+    else:
+        print(
+            "You haven't specified a partitioning algorihm in the general section of the setup.json. This defaults to meshfree, which uses a non-deterministic k-means clusterting. Partitionings will be non-reproducible."
```

Comment:
From what we have learned here, topology is also non-deterministic, so what's the point of the message?

Reply:
The seed is fixed now on develop. So it should be deterministic. However, PR #224 will make both deterministic, so there is no reason to keep this around.